profiling/src/php_ffi.c

```diff
@@ -146,3 +146,68 @@ zend_execute_data* ddog_php_prof_get_current_execute_data()
 {
     return EG(current_execute_data);
 }
+
+#if CFG_STACK_WALKING_TESTS
+zend_execute_data* ddog_php_test_create_fake_zend_execute_data(int depth) {
+    if (depth <= 0 || depth > 99) {
```

Comment:
Minor: There is probably value in allowing the test to work for more extreme stack depths.